webpack.config.js

```diff
@@ -2,18 +2,20 @@ const nodeExternals = require('webpack-node-externals');
 const path = require("path");
 const webpack = require('webpack');
 const { CleanWebpackPlugin } = require("clean-webpack-plugin")
-const CopyWebpackPlugin = require("copy-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin"); 
 
 module.exports = (env, argv) => {
     return {
+        
         entry: {app: './index.ts'},
+
         externalsPresets: { node: true },
         context: __dirname,
 
         externals: [nodeExternals()],
         output: {
             path: path.join(__dirname, "dist"),
-            filename: "atm-v-01.js"
+            filename: "active-task-v-01.js"
```

Comment:
@oscarmild Please add some more plugins for reducing output file

Reply:
@fromgopi it will be in other branch